homeassistant/components/deconz/strings.json

```diff
@@ -23,15 +23,15 @@
       }
     },
     "error": {
-      "no_key": "Couldn't get an API key"
+      "invalid_api_key": "[%key:common::config_flow::error::invalid_api_key%]"
     },
     "abort": {
       "already_configured": "Bridge is already configured",
       "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
       "no_bridges": "No deCONZ bridges discovered",
       "no_hardware_available": "No radio hardware connected to deCONZ",
       "not_deconz_bridge": "Not a deCONZ bridge",
-      "one_instance_only": "Component only supports one deCONZ instance",
+      "single_instance_allowed": "[%key:common::config_flow::error::single_instance_allowed%]",
```

Comment:
I'm not sure about it, but following the previous comments on #40578 we should adopt the standards